docs/feature_led_indicators.md

```diff
@@ -22,6 +23,10 @@ Two deprecated functions that provide the LED state as `uint8_t`:
 * `uint8_t led_set_user(uint8_t usb_led)`
 * `uint8_t host_keyboard_leds()`
 
+## `info.json` Options
+Information about using data driven configuration specified in the `info.json` file can be found [at the following link](reference_info_json.md#led-indicators). 
```

Comment:
We are still looking at the best way to document the data-driven stuff, it will probably not be like this.